### PR TITLE
Update documentation to allow for 3 TFE nodes

### PR DESCRIPTION
**Pull request by amy-hashi**
Add a section to the TFE Active/Active page around have a three node setup. Also update other references to two nodes being the maximum supported.

### CODE REVIEW

File: content/enterprise/install/automated/active-active.mdx
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:255]:
```suggestion
As you scale Terraform Enterprise beyond two nodes, you may be adding additional stress to your network and dramatically increasing the number of API calls made in your cloud account. Each support cloud has its own default limits and processes by which those limits might be increased. Please refer to the documentation for the cloud hosting your installation.
```

File: content/enterprise/install/automated/active-active.mdx
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:264]: Moved that to above the links :-) 

```suggestion
```

File: content/enterprise/install/automated/active-active.mdx
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:268]:
```suggestion
Instead of scaling Terraform Enterprise beyond two or three nodes, you can use [Terraform Cloud Agents](https://www.terraform.io/enterprise/admin/agents-on-tfe). Terraform Cloud Agents can run in other regions, other clouds, and even private clouds. Agents poll Terraform Enterprise for work and then Terraform plans and applies will run on the target system that has the agent executable installed. This has a much smaller impact on the Terraform Enterprise servers than running Terraform locally.
```

File: content/enterprise/install/automated/active-active.mdx
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:266]:
```suggestion
#### Terraform Cloud Agents - Alternative Solution
```

**Review by laurapacilio — APPROVED** [review]
Approving so I don't block :) A couple of wording suggestions for you! Feel free to tweak any that materially change the meaning in a way you don't like.  Thank you for adding this!!

**Comment by laurapacilio** [comment]
Also @amy-hashi We have a deploy failure for the Vercel preview. Please don't merge this with a deploy failure! I'm hoping it may resolve itself as you continue to edit, but if continues to be failed, we need to talk to web dev before merging :)

File: content/enterprise/install/automated/active-active.mdx
Comment (jmariedee) [inline content/enterprise/install/automated/active-active.mdx:233]: Here we spell out "three" and "two" whereas earlier in the document we use digits. This is super nit-picky, but I'm curious if we have a style guide rule to turn to here?

File: content/enterprise/install/automated/active-active.mdx
Comment (jmariedee) [inline content/enterprise/install/automated/active-active.mdx:264]: Great idea including this!

File: content/enterprise/install/automated/active-active.mdx
Comment (laurapacilio) [reply content/enterprise/install/automated/active-active.mdx:233]: such a good point!! I pulled this from our internal style guide: 

"For more formal writing, write out “one” through “nine” and write 10 and above as numerals.
For more technical writing in sentences with technical terms, write most numbers as numerals for ease of reading. Spell out “one” if it’s not used in tandem with another number (like in a range 1 to 4 for example) in the sentence."

As this is technical content, I believe we should write them all as numerals. Let me know if you disagree!

Comment (amy-hashi) [reply content/enterprise/install/automated/active-active.mdx]: I have no objection. Done

**Pull request merged**
